mypy_django_plugin/transformers/models.py

```diff
@@ -297,12 +302,26 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
 
 
 class AddRelatedManagers(ModelClassInitializer):
+    def get_reverse_manager_info(self, model_info: TypeInfo, derived_from: str) -> Optional[TypeInfo]:
+        manager_fullname = helpers.get_django_metadata(model_info).get("reverse_managers", {}).get(derived_from)
+        if not manager_fullname:
+            return None
+
+        symbol = self.api.lookup_fully_qualified_or_none(manager_fullname)
+        if symbol is None or not isinstance(symbol.node, TypeInfo):
+            return None
+        return symbol.node
+
+    def set_reverse_manager_info(self, model_info: TypeInfo, derived_from: str, fullname: str) -> None:
```

Comment:
Please, make sure that we use both flags here: https://github.com/python/mypy/blob/0cec4f7a2e2c569b28f0ced916edd57903e57b7a/mypy/nodes.py#L3173-L3186

Reply:
Hm, yeah. I'm guessing that we'd like to have `plugin_generated=True` and `no_serialize=True` set for both the new manager type we create and the reverse manager node we insert on the related model. Does that sound reasonable?

Reply:
Also, I'm not sure if it's a bit too optimistic, in regards to cache, with the adjustment I did to look if the reverse manager attribute is already inserted into the related model's symbol table

https://github.com/typeddjango/django-stubs/pull/825/files#diff-fcf5f5a98547e722d4134f8bd410d2f21a686877f1fdcfac421373edf13ff67eR322-R324